ntc_templates/templates/cisco_asa_show_running-config_access-group.textfsm

```diff
@@ -0,0 +1,9 @@
+Value ACCESS_LIST (\S+)
+Value INTERFACE (\S+)
+
+
```

Comment:
Minor item, we can drop one of the two blank lines between capture group definition and the Start state.

```suggestion
```